pkg/stdlib/html/pagination.go

```diff
@@ -92,11 +92,17 @@ func (i *PagingIterator) Next(ctx context.Context) (core.Value, core.Value, erro
 		return values.ZeroInt, values.ZeroInt, nil
 	}
 
-	if !i.document.ExistsBySelector(ctx, i.selector) {
+	exists, err := i.document.ExistsBySelector(ctx, i.selector)
+
+	if err != nil {
+		return values.ZeroInt, values.ZeroInt, err
```

Comment:
Maybe return `values.None` instead `values.ZeroInt`?

Reply:
It does not really matter, since an execution gets terminated because of the returned error.

Reply:
To be consistent I'm suggest to return `values.None` every time error occured.